scripts/parse-arguments.sh

```diff
@@ -111,6 +112,10 @@ for argument in "$@"; do
     EXTERNAL_LIBRARIES+=("libx264")
     FFMPEG_GPL_ENABLED=true
     ;;
+  --enable-libbluray | -libbluray)
```

Comment:
Please use just `-bluray` for the short version.

Reply:
ok